threadedcomments/models.py

```diff
@@ -143,35 +181,61 @@ class ThreadedComment(models.Model):
 
     It also includes two Managers: ``objects``, which is the same as the normal
     ``objects`` Manager with a few added utility functions (see above), and
-    ``public``, which has those same utility functions but limits the QuerySet to
-    only those values which are designated as public (``is_public=True``).
+    ``public``, which has those same utility functions but limits the QuerySet
+    to only those values which are designated as public (``is_public=True``).
     """
     # Generic Foreign Key Fields
     content_type = models.ForeignKey(ContentType)
     object_id = models.PositiveIntegerField(_('object ID'))
     content_object = generic.GenericForeignKey()
 
     # Hierarchy Field
-    parent = models.ForeignKey('self', null=True, blank=True, default=None, related_name='children')
+    parent = models.ForeignKey(
+        'self',
+        null=True,
+        blank=True,
+        default=None,
+        related_name='children',
+    )
 
     # User Field
     user = models.ForeignKey(User)
 
     # Date Fields
-    date_submitted = models.DateTimeField(_('date/time submitted'), default = datetime.now)
-    date_modified = models.DateTimeField(_('date/time modified'), default = datetime.now)
-    date_approved = models.DateTimeField(_('date/time approved'), default=None, null=True, blank=True)
+    date_submitted = models.DateTimeField(
+        _('date/time submitted'),
+        default=datetime.now,
+    )
+    date_modified = models.DateTimeField(
+        _('date/time modified'),
+        default=datetime.now,
+    )
+    date_approved = models.DateTimeField(
+        _('date/time approved'),
+        default=None,
+        null=True,
+        blank=True,
+    )
 
     # Meat n' Potatoes
     comment = models.TextField(_('comment'))
-    markup = models.IntegerField(choices=MARKUP_CHOICES, default=DEFAULT_MARKUP, null=True, blank=True)
+    markup = models.IntegerField(
+        choices=MARKUP_CHOICES,
+        default=DEFAULT_MARKUP,
+        null=True,
+        blank=True,
+    )
 
     # Status Fields
-    is_public = models.BooleanField(_('is public'), default = True)
-    is_approved = models.BooleanField(_('is approved'), default = False)
+    is_public = models.BooleanField(_('is public'), default=True)
+    is_approved = models.BooleanField(_('is approved'), default=False)
 
     # Extra Field
-    ip_address = models.IPAddressField(_('IP address'), null=True, blank=True)
+    ip_address = models.GenericIPAddressField(
```

Comment:
i feel like I've asked this before, but this doesn't cause it to think it needs a migration, does it?